nucypher/blockchain/eth/sol/source/contracts/StakingEscrow.sol

```diff
@@ -543,6 +543,11 @@ contract StakingEscrow is Issuer, IERC900History {
         external
     {
         require(msg.sender == address(workLock));
+        StakerInfo storage info = stakerInfo[_staker];
+        if (!info.flags.bitSet(WIND_DOWN_INDEX) && info.subStakes.length == 0) {
```

Comment:
info.subStakes.length == 0
what is the purpose of this condition?

Reply:
To check if it first deposit, because if not - then suppose that staker already set wind down true/false as wants

Reply:
That's toughtful!